index.html

```diff
@@ -9,23 +9,32 @@
       href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
     />
     <link rel="stylesheet" href="style.css" />
-    <title>responsive website portfolio</title>
+    <title>responsive-website-portfolio</title>
   </head>
   <body>
     <header>
-      <div class="my-logo container">
-        <a href="#" class="logo">Rejoice</a>
-
-        <ul class="nav-elements">
-          <li class="element-nav"><a href="#">Portfolio</a></li>
-          <li class="element-nav"><a href="#">About</a></li>
-          <li class="element-nav"><a href="#">Contact</a></li>
-        </ul>
-        <nav class="menu-bar">
-          <i class="fas fa-bars"></i>
-        </nav>
-      </div>
+      <nav class="nav-bar">
+          <div class="logo logo-rotated">
+            <a href="#">Rejoice Evans</a>
+          </div>
+          <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+          <ul class="nav-links">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
```

Comment:
- Optional
When I click on the navbar links in the desktop mode, they don't display anything, instead, it would be better if I click say on the portfolio link and it takes me to the portfolio section.